ts/src/base/Exchange.ts

```diff
@@ -1699,11 +1699,159 @@ export default class Exchange {
+                // the structure is like `{ mainnetCurrencyCode : {networkCodeProtocol : networkCodeMainnet} }`
                 'ETH': { 'ERC20': 'ETH' },
                 'TRX': { 'TRC20': 'TRX' },
                 'CRO': { 'CRC20': 'CRONOS' },
+                // for rest of tokens, if user provides "currencyCode=XYZ & networkCode=ETH" then we replace networkCode with "ERC20"
+                '*': {
+                    'ETH': 'ERC20', // { networkCodeMainnet : networkCodeProtocol }
+                    'TRX': 'TRC20',
+                    'CRONOS': 'CRC20',
+                },
             },
+            // this list is for common reserved CCXT unified network codes, the primary and supported secondary unified network codes
+            'unifiedNetworkCodesAndAliases': {
+                'BSC': 'BEP20',
+                'HECO': 'HRC20',
+                'TRON': 'TRC20',
+                // 'BNB': 'BEP2', // BNB is risky for unification, as some exchanges are undeliberately calling 'BNB' network for BINANCE smart chain
+                'CARDANO': 'ADA',
+                'DOGECOIN': 'DOGE',
+                'SOLANA': 'SOL',
+                'POLYGON': 'MATIC',
+                'COSMOS': 'ATOM',
+                'POLKADOT': 'DOT',
+                'RIPPLE': 'XRP',
+                'STELLAR': 'XLM',
+                'ALGORAND': 'ALGO',
+                'CHILIZ': 'CHZ',
+                'ZILLIQA': 'ZIL',
+                'ECASH': 'XEC',
+                'ZCASH': 'ZEC',
+                'KUSAMA': 'KSM',
+                'MOONBEAM': 'GLMR',
+                'MOONRIVER': 'MOVR',
+                'INJECTIVE': 'INJ',
+                'SYMBOL': 'XYM',
+                'ONTOLOGY': 'ONT',
+                'ONTOLOGY_GAS': 'ONG',
+                'NEBULAS': 'NAS',
+                'ELROND': 'EGLD',
+                'TELOS': 'TLOS',
+                // ARBITRUM_ONE: ARB_ONE,
+                // ARBITRUM_NOVA: ARB_NOVA,
+                // AVALANCHE_X: AVAX_X,
+                // AVALANCHE_C: AVAX_C,
+                'SOLAR': 'SXP',
+                'FILECOIN': 'FIL',
+                'KLAYTN': 'KLAY',
+                'PROTON': 'XPR',
+                'HARMONY': 'ONE',
+                'BITCOIN_LIGHTNING': 'BTCLN',
+                'KARURA': 'KAR',
+                'KADENA': 'KDA',
+                'TEZOS': 'XTZ',
+                // smaller networks
+                'POLKADEX': 'PDEX',
+                'BITTORRENT': 'BTT',
+                'VECHAIN': 'VET',
+                'BITSHARES': 'BTS',
+                'EVERSCALE': 'EVER',
+                'HEDERA': 'HBAR',
+                'ACALA': 'ACA',
+                'ARWEAVE': 'AR',
+                'BITCOINX': 'BCX',
+                'APTOS': 'APT',
+                'DIGIBYTE': 'DGB',
+                'SUPERBITCOIN': 'SBTC',
+                'THUNDERCORE': 'TT',
+                'DECRED': 'DCR',
+                'DECRED_NEXT': 'DCRN',
```

Comment:
they are not "major" codes, they are just "readable alias list" so that's why I've left them with lower underscore to be readable to public/readers (also many exchange use it)